inst/validation/utils-validation.R

```diff
@@ -0,0 +1,148 @@
+#' Setting up the validation
+#'
+#' 1. Add package_name
```

Comment:
check comments and update if necessary

Reply:
No need to update, as mentioned qc for now works with local copies in each repo.